twoliter/src/test/mod.rs

```diff
@@ -4,7 +4,7 @@ This directory and module are for tests, test data, and re-usable test code. Thi
 be compiled for `cfg(test)`, which is accomplished at its declaration in `main.rs`.
 
 !*/
-
+pub mod cargo_make;
```

Comment:
I think it can be private.

```suggestion
mod cargo_make;
```